src/crypto/crypto.js

```diff
@@ -28,23 +28,29 @@ import publicKey from './public_key';
 import * as cipher from './cipher';
 import { getRandomBytes } from './random';
 import ECDHSymkey from '../type/ecdh_symkey';
+import ShortByteString from '../type/short_byte_string';
+import hash from './hash';
+import config from '../config';
 import KDFParams from '../type/kdf_params';
+import { SymAlgoEnum, AEADEnum, HashEnum } from '../type/enum';
 import enums from '../enums';
 import util from '../util';
 import OID from '../type/oid';
 import { Curve } from './public_key/elliptic/curves';
+import crypto from './';
```

Comment:
Rather than importing the entire crypto module, please import the specific submodule you need (e.g. `./mode`)